docs/docsite/rst/plugins/plugins.rst

```diff
@@ -25,6 +25,7 @@ This section covers the various types of plugins that are included with Ansible:
    shell
    strategy
    vars
+   process
```

Comment:
To preserve the alphabetization, `process` should slot in under `lookup` and above `shell`.